embeddings/bedrock/bedrock_test.go

```diff
@@ -0,0 +1,34 @@
+package bedrock_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tmc/langchaingo/embeddings/bedrock"
+)
+
+func TestEmbedQuery(t *testing.T) {
+	t.Parallel()
+	model, err := bedrock.NewBedrock(bedrock.WithModel(bedrock.ModelTitanEmbedG1))
+	if err != nil {
+		t.Skip("Skipping test, failed to initialize bedrock model")
+	}
+
+	_, err = model.EmbedQuery(context.Background(), "hello world")
+
+	require.NoError(t, err)
+}
+
+func TestEmbedDocuments(t *testing.T) {
+	t.Parallel()
+	model, err := bedrock.NewBedrock(bedrock.WithModel(bedrock.ModelCohereEn))
+	if err != nil {
+		t.Skip("Skipping test, failed to initialize bedrock model")
+	}
```

Comment:
This is not working I think, it's picking credentials from somewhere. I need some suggestions. Maybe some new environment variable to be explicitly enabled?

Reply:
Added env `TEST_AWS` if you want to test with AWS default credentials chain.

Reply:
ehh yeah I'd prefer to check for env var poppulation rather than just skipping on init failure.